File: 0009-palindrome-number/0009-palindrome-number.py
class Solution(object):
    def isPalindrome(self, x):
        real = x
        rev = 0
        while x > 0:
            rem = x % 10
            x = x / 10
            rev = ( rev * 10 ) + rem
        if real == rev:
            return True
        return False
        

File: 0009-palindrome-number/0009-palindrome-number.cpp
class Solution {
public:
    bool isPalindrome(int x) {
        if (x<0) return false;
        if (x == 0) 
            return true;
        int s = x;
        long currentDigit = 0;
        long rev = 0;
        while(s>0){
            currentDigit = s % 10;
            rev = rev*10 + currentDigit;
            s = s/10;
        }
        
        if (rev==x) return true;
        return false;
    }
};